desktop/cmp/panel/PanelModel.ts

```diff
@@ -35,8 +36,8 @@ export interface PanelConfig {
     /** Can panel be collapsed, showing only its header? */
     collapsible?: boolean;
 
-    /** Default size (in px) of the panel. */
-    defaultSize?: number;
+    /** Default size (in px or %) of the panel. */
+    defaultSize?: number | string;
```

Comment:
Would like to make this a bit more clear - are these the supported formats?
1) Pixels, as a number
2) Pixels, as a string "Npx"
3) Percent, as a string "N%"

Would like to ensure it is fully clear that the number variant is _only_ pixels and if the string version is _only_ for percents, or if it's for either unit. 

With these properties that take flexible types I think it's worth being as clear as possible, especially since it's asymmetrical where one unit could be written in either type (maybe) and the other only in one type.